base/testfiles-TU/github-1100.tlg

```diff
@@ -0,0 +1,2 @@
+This is a generated file for the LaTeX2e validation system.
+Don't change this file in any respect.
```

Comment:
is the test not showing what it is supposed to show, ie some typeout data?

Reply:
forget it that is the pdftex one ... got it

Reply:
actually xetex, I put this in the -TU config to cut down on non-luatex runs, I was surprised we don't have a luatex-only config but so far we have avoided that and just had empty tests for non lua engines eg testfiles/tlb4445.lvt... Could add a `config-lua` and remove the `\ifx\directlua\undefined` test as we have a few lua-only tests and likely to get more I suspect

```
$ grep -l ifx.directlua testfiles*/*lvt
testfiles-TU/github-0282.lvt
testfiles-TU/github-1100.lvt
testfiles-disabled/tlb-early-expl3.lvt
testfiles/github-0591.lvt
testfiles/tlb-callbacks-001.lvt
testfiles/tlb-callbacks-002.lvt
testfiles/tlb-extraprimitives-001.lvt
testfiles/tlb-latexrelease-rollback-003-often.lvt
testfiles/tlb-ltluatex-001.lvt
testfiles/tlb-luacmd-001.lvt
testfiles/tlb-mathcallbacks-001.lvt
testfiles/tlb-mathcallbacks-002.lvt
testfiles/tlb4445.lvt
```